### PR TITLE
Fix/147 update chrome launch flags

### PR DESCRIPTION
**Pull request by inancgumus**
Please see: #147

### CODE REVIEW

File: chromium/browser_type_test.go
Comment (imiric) [inline chromium/browser_type_test.go:46]: We can rely on the `testify` API to do these checks, so this can be `require.True(t, ok)`. You can use `Truef()` if you want the message as well.

I typically use `require` instead of `assert` if the following code depends on the assertion passing.

File: chromium/browser_type_test.go
Comment (imiric) [inline chromium/browser_type_test.go:50]: Same here, this can be `require.True(t, ok)`.

File: chromium/browser_type_test.go
Comment (imiric) [inline chromium/browser_type_test.go:53]: `assert.Falsef(t, flag, ...)`

File: chromium/browser_type_test.go
Comment (imiric) [inline chromium/browser_type_test.go:61]: `assert.Falsef(t, flag, ...)`

File: chromium/browser_type_test.go
Comment (imiric) [inline chromium/browser_type_test.go:95]: `assert.Contains(t, f, "enable-use-zoom-for-dsf")`

**Review by imiric — CHANGES_REQUESTED** [review]
LGTM, just some testing suggestions.

I haven't looked at Playwright to compare, so @robingustafsson should approve this before merging.

Just for reference, can you link to the Playwright code where these are set? Both in the PR description and in the commit message?

File: chromium/browser_type_test.go
Comment (inancgumus) [reply chromium/browser_type_test.go:46]: I'm not a fan 3rd party pkgs while we have `testing.T`. I'm not sure what value it brings. It's like using a pkg for skip writing `if err != nil`. It steals the expressiveness from code.

But.... Sure.

Comment (imiric) [reply chromium/browser_type_test.go:46]: It standardizes the way assertions are made, and makes tests more readable precisely by avoiding repetitive checks. I thought you were a fan of readability and abstractions? :)

Besides, it's a pretty widely adopted package, there's no harm in depending on it.

Comment (inancgumus) [reply chromium/browser_type_test.go:46]: `testing.T` is already standardized, and we have good conventions around it. Anybody who knows how to write idiomatic Go code already knows how to use it effectively.

I guess you misunderstood me before. I'm a fan of readability, but not useless abstractions. I believe in abstractions only if they don't take away any expressiveness from code. `if a != b { t.Errorf(...) }` is as expressive as `if err != nil`.

**Review by imiric — APPROVED** [review]

**Comment by imiric** [comment]
> @robingustafsson should approve this before merging.

??

**Comment by inancgumus** [comment]
Ah, sorry, I thought it was ready to merge after you approved the changes.

**Pull request merged**
